.github/workflows/maven_test.yml

```diff
@@ -115,6 +115,7 @@ jobs:
       HIVE_PROFILE: ${{ matrix.hive }}
       SPARK_LOCAL_IP: localhost
       GITHUB_PREV_SHA: ${{ github.event.before }}
+      SKIP_SPARK_RELEASE_VERSIONS: ${{ fromJSON(inputs.envs).SKIP_SPARK_RELEASE_VERSIONS }}
```

Comment:
This option should not be needed, the `Run tests` section has already been deserialized for env

https://github.com/apache/spark/blob/0b8e0998c67331bc82428d807476584f3d21f934/.github/workflows/maven_test.yml#L172-L173